stock_inventory_revaluation/wizards/stock_inventory_revaluation_mass_post.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, models, _, exceptions
+
+
+class StockInventoryRevaluationMassPost(models.TransientModel):
+
+    _name = 'stock.inventory.revaluation.mass.post'
+    _description = 'Post multiple inventory revaluations'
+
+    @api.model
+    def default_get(self, fields):
```

Comment:
Hi @jbeficent , coverage score decreased due this method. No test cover this method. IMO you could remove this method. I do not see the usage of this method. Am i miss something? Would care to explain why you need this method.

I create a PR to your branch to address api.one comment. Would you kindly merged it.

Many thanks